exe/puppet-query.cc

```diff
@@ -60,7 +70,7 @@ main(int argc, char **argv) {
             po::store(parsed, vm);
 
             if (vm.count("help")) {
-                help(global_options);
+                help(global_options, nowide::cout);
```

Comment:
This made me pause, but the cout behaviour is correct when people ask for help, a comment to this affect reminding everyone might stop people from pausing :-).